h2o-r/h2o-package/R/models.R

```diff
@@ -853,12 +856,19 @@ h2o.staged_predict_proba <- staged_predict_proba.H2OModel
 #' h2o.predict_contributions(prostate_gbm, prostate)
 #' }
 #' @export
-predict_contributions.H2OModel <- function(object, newdata, ...) {
+predict_contributions.H2OModel <- function(object, newdata, output_format = c("Original", "Compact"), ...) {
```

Comment:
I don't have strong opinion about this but in explain, we are trying to have lowercase enums (inspired by tidyverse[1]). We should probably have some coding style guide to keep it in sync.

[1] https://style.tidyverse.org

Reply:
Make sense, I followed the example in leaf node assignment

```
   predict_leaf_node_assignment.H2OModel <- function(object, newdata, type = c("Path", "Node_ID"), ...) {
```
we will have to change it there as well

Reply:
fixed, I made a jira for predict_leaf_node_assignment https://h2oai.atlassian.net/browse/PUBDEV-8055